devices.js

```diff
@@ -8424,8 +8424,8 @@ const devices = [
             await configureReporting.brightness(endpoint);
             await readEletricalMeasurementPowerConverterAttributes(endpoint);
             await configureReporting.activePower(endpoint);
-            await configureReporting.rmsCurrent(endpoint, {min: 5, change: 5});
-            await configureReporting.rmsVoltage(endpoint, {min: 5});
+            await configureReporting.rmsCurrent(endpoint, {min: 10, change: 10});
+            await configureReporting.rmsVoltage(endpoint, {min: 10});
```

Comment:
It would be better to rely more on `change` to avoid reporting when the value stays +- the same. To figure out what value to use start with e.g. `10` and see if values are reported every 10 seconds (because that is the min interval you specified). If so try with `100`, `1000`, etc. Once you found the value were it doesn't report every 10 seconds anymore see if it still reports when attaching a load. I would recommend the same for `activePower`.

Reply:
I completely agree with you and I would definitely prefer that, but the problem is `voltageRms` values just jump around... I got a sample log (full [here](https://pastebin.com/raw/BSz2whVr), and filtered by only voltageRms [here](https://pastebin.com/raw/HG6wJTy6)) of this device using the above configuration (it was off, then I turn it on for a few seconds, then off again), you can easily see on the top that the values are just jumping around (this is probably due to the fact this dimmer does not require a neutral wire to work)

Reply:
To be honest, I have my doubts that voltage RMS is actually being calculated correctly on this device...

Reply:
Clear! In that case this can be merged.